test/server.test.js

```diff
@@ -4,14 +4,179 @@ import request from 'supertest';
 import {app} from '../server';
 
 describe('Server', () => {
-  it('should respond with status 200 when GET is sent to /template', (done) => {
-    request(app)
-      .get('/template')
-      .expect(200)
-      .end((err, res) => {
-        if (err) return done(err);
-        done();
-      });
-  }); 
+  describe('Template path', () => {
```

Comment:
I wonder if we still need the template routes and files. We can chat about this later.

Reply:
Yeah, I should probably remove it